netdata-installer.sh

```diff
@@ -1318,11 +1318,30 @@ should_install_ebpf() {
   return 0
 }
 
+remove_old_ebpf() {
+  if [ -f "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/ebpf_process.plugin" ]; then
+    echo >&2 "Removing alpha eBPF collector."
+    rm -rf "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/ebpf_process.plugin" 
+  fi
+
+  if [ -f "${NETDATA_PREFIX}/usr/lib/netdata/conf.d/ebpf_process.conf" ]; then
+    echo >&2 "Removing alpha eBPF stock file"
+    rm -rf "${NETDATA_PREFIX}/usr/lib/netdata/conf.d/ebpf_process.conf" 
```

Comment:
same as above

Reply:
like this previous this was a safe choice.